impeller/core/sampler_descriptor.h

```diff
@@ -21,6 +21,14 @@ struct SamplerDescriptor final : public Comparable<SamplerDescriptor> {
   SamplerAddressMode height_address_mode = SamplerAddressMode::kClampToEdge;
   SamplerAddressMode depth_address_mode = SamplerAddressMode::kClampToEdge;
 
+  /// The minimum lod of the texture will be clamped to this value.
+  ///
+  /// This value is ignored with a mip_filter of MipFilter::kBase.
+  ///
+  /// Defaults to 0, the base mip level. Only supported on Vulkan and
+  /// Metal backends.
+  size_t min_lod = 0;
```

Comment:
We may not actually need this. Since the downsample is implied by the blur subpasses.